packages/frontend/core/src/components/pure/cmdk/data.tsx

```diff
@@ -168,21 +169,15 @@ export const pageToCommand = (
   const commandLabel = label || {
     title: title,
   };
-  const escapedTitle = escape(title);
 
   return {
     id: page.id,
     label: commandLabel,
     // hack: when comparing, the part between >>> and <<< will be ignored
     // adding this patch so that CMDK will not complain about duplicated commands
     value:
-      escapedTitle +
-      valueWrapperStart +
-      page.id +
-      '.' +
-      category +
-      valueWrapperEnd,
-    originalValue: escapedTitle,
+      title + valueWrapperStart + page.id + '.' + category + valueWrapperEnd,
```

Comment:
I am curious why the existing solution does not work. Can you please explain?

Reply:
It appears to work well for searching existing articles, however, in reality, when you solely search for a double quote, the results turn up entirely empty. Also, when I attempted to alter the value of the ensuing command using the same method, the problem arose. I am incapable of creating any pages using the command.

Nonetheless, this issue only arises when inputting double quotations. Other special symbols seem to be managed efficiently. I attempted to escape the value of the input and command item, but it necessitated a change in the upstream repository, hence, I abandoned this idea.

```
...
      if (results.every(command => command.originalValue !== query)) {
        const escapedQuery = escape(query);
        results.push({
          id: 'affine:pages:create-page',
          label: t['com.affine.cmdk.affine.create-new-page-as']({
            keyWord: query,
          }),
          value: 'affine::create-page' + escapedQuery, 
          category: 'affine:creation',
          run: async () => {
            const page = pageHelper.createPage();
            await page.waitForLoaded();
            pageMetaHelper.setPageTitle(page.id, query);
          },
          icon: <PageIcon />,
        });
...
```
<img width="729" alt="image" src="https://github.com/toeverything/AFFiNE/assets/102217452/9dffa64c-4ecf-4e52-bbb6-6acdbde17626">
<img width="706" alt="image" src="https://github.com/toeverything/AFFiNE/assets/102217452/e5c922f9-8368-4172-aac3-07266d329325">

Reply:
Sounds fair enough.
We may not need to worry about too much on how CMDK library for searching & sorting later, since implementing an in-house solution for that is already planned :/